modular_skyrat/modules/modular_items/code/modular_clothing.dm

```diff
@@ -1,45 +0,0 @@
-/*
```

Comment:
not only were none of the items in this file available to get, and very crummily sprited, they all defied the master-file clothing icon rules!